.github/workflows/vast.yml

```diff
@@ -133,25 +133,25 @@ jobs:
           echo ::set-env name=GITHUB_HEAD_REF_SLUG::"$(slug_ref "$GITHUB_HEAD_REF")"
           echo ::set-env name=GITHUB_BASE_REF_SLUG::"$(slug_ref "$GITHUB_BASE_REF")"
 
-      - name: Fetch ccache Cache
+      - name: Fetch ccache Cache (pull_request)
+        if: github.event_name == 'pull_request'
         uses: actions/cache@v2
         with:
           path: .ccache
           key: ccache-${{ matrix.os.tag }}-${{ matrix.os.compiler }}-${{ matrix.configure.tag }}-${{ env.GITHUB_HEAD_REF_SLUG }}-${{ github.sha }}
```

Comment:
Sorry for being late to the party:
According to [the documentation](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables): `GITHUB_HEAD_REF` and `GITHUB_BASE_REF` are only set in forked repos?

It would also be great if these new env-variables would be logged to the console here.

Reply:
They are logged to the console implicitly.

Reply:
And "forked repos" basically means they're only available in `pull_request` events.